File: exercicio03/java/Figura2.java
package exercicio03.java;

public class Figura2 {
  public double coordX;
  public double coordY;

  public Figura2(double x, double y) {
    this.coordX = x;
    this.coordY = y;
  }

  public void desenhar() {
    System.out.println("Desenho das coordenadas (" + this.coordX + ", " + this.coordY + ")");
  }

  public void pintar() {
    System.out.println("Pintando os pontos das coordenadas (" + this.coordX + ", " + this.coordY + ")");
  }

  public void imprimi() {
    System.out.printf("Coordenadas_da_Figura : (%f, %f)%n ", this.coordX, this.coordY);
  }
}

File: exercicio03/python/figura2.py
class Figura2:
    coordenadaX = 0.0
    coordenadaY = 0.0

    def __init__(self, x: float, y: float) -> None:
        self.coordenadaX = x
        self.coordenadaY = y

    def desenhar(self):
        print(
            f"Desenho das coordenadas ({self.coordenadaX}, {self.coordenadaY})")

    def pintar(self):
        print(
            f"Pintando os pontos das coordenadas ({self.coordenadaX}, {self.coordenadaY})")

    def imprimi(self):
        print(f"Coordendas: ({self.coordenadaX}, {self.coordenadaY})")
